demo-app/src/lib/synchronous-uqi-query.ts

```diff
@@ -0,0 +1,64 @@
+import sources from '@/lib/uqi-sources'
```

Comment:
This file doesn't appear to be used. Did you intend to add this? Maybe you wanted to change the api's functionality to use this lib?

Reply:
It's used on line 38: https://github.com/open-truss/open-truss/pull/90/files#diff-0d36a671fa5661d4608140df44061a3f01682b4f6943bc4eb5b16248db76cfd6R38